kind.sh

```diff
@@ -47,10 +47,35 @@ main() {
 
     parse_command_line "$@"
 
-    install_kind
-    install_kubectl
-    create_kind_cluster
+    if [[ ! -d "$RUNNER_TOOL_CACHE" ]]; then
+        echo "Cache directory '$RUNNER_TOOL_CACHE' does not exist" >&2
```

Comment:
Shouldn't we make the directory here if it doesn't exist instead of exiting? Quick review from my phone so I might be missing something, just checking

Reply:
Nope, that's expected to be present

Reply:
I guess this could be an issue for a local runner. But I'd say it's not set up properly then.

Reply:
worked fine for chart-testing-action on self-hosted runners - so I believe the runner defines these vars and are inherited by spawned envs.